From baa2ebd111d4f8df66254bc94ee79aeaae2a3f3e Mon Sep 17 00:00:00 2001
From: Petr Uzel <petr.uzel@suse.cz>
Date: Mon, 15 Oct 2012 10:31:54 +0200
Subject: [PATCH 38/89] partprobe: remove --no-update long option

It was deprecated and scheduled for removal in 2009.
* partprobe/partprobe.c (long_options): Remove "no-update" entry.
---
 partprobe/partprobe.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/partprobe/partprobe.c b/partprobe/partprobe.c
index 8bccc4f..b8dca5e 100644
--- a/partprobe/partprobe.c
+++ b/partprobe/partprobe.c
@@ -57,9 +57,6 @@
 
 static struct option const long_options[] =
   {
-    /* Note: the --no-update option is deprecated, and deliberately
-     * not documented.  FIXME: remove --no-update in 2009. */
-    {"no-update", no_argument, NULL, 'd'},
     {"dry-run", no_argument, NULL, 'd'},
     {"summary", no_argument, NULL, 's'},
     {"help", no_argument, NULL, 'h'},
-- 
1.8.5.3

